### PR TITLE
Novica veljanoski question

### PR DESCRIPTION
**Pull request by 173038**

### CODE REVIEW

File: project-novica/demo/src/main/java/com/example/demo/controller/AuthenticationController.java [hunk 1]
```diff
@@ -72,22 +106,27 @@ public String getLoginPage(Model model)
...
+         return ResponseEntity.ok("success");
+     }
+     else
+     {
+         return ResponseEntity.badRequest().body(res);
+     }
+ }
+
+    @RequestMapping(path = "/question", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    public List<Question> listQuestion()
```
Comment (dejannastevski-armedia) [inline project-novica/demo/src/main/java/com/example/demo/controller/AuthenticationController.java]: Why do you have addQuestion and listQuestions here?

File: project-novica/demo/src/main/java/com/example/demo/controller/HomeController.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+@Controller
+public class HomeController
+{
+
+    @Autowired
+    QuestionService questionService;
+
+    @RequestMapping(path = "/home", method = RequestMethod.POST)
+    @ResponseBody
+    public ResponseEntity<String> addQuestion(@RequestBody QuestionDTO questionDTO)
```
Comment (dejannastevski-armedia) [inline project-novica/demo/src/main/java/com/example/demo/controller/HomeController.java]: And you have again addQuestion here in a controller named HomeController?

File: project-novica/demo/src/main/java/com/example/demo/service/impl/AuthenticationServiceImpl.java [hunk 3]
```diff
@@ -138,17 +137,17 @@ public void deleteUserById(Long id)
         }
         if (!checkPassword(password))
         {
-            //throw new InvalidArgumentException("Enter valid password");
+            // throw new InvalidArgumentException("Enter valid password");
```
Comment (dejannastevski-armedia) [inline project-novica/demo/src/main/java/com/example/demo/service/impl/AuthenticationServiceImpl.java:140]: Remove these lines that are commented.

File: project-novica/demo/src/main/java/com/example/demo/service/impl/QuestionServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    }
+
+    @Override
+    public void saveQuestion(Question question)
+    {
+        questionRepository.save(question);
+    }
+
+    @Override
+    public String validateQuestion(String question, String title)
```
Comment (dejannastevski-armedia) [inline project-novica/demo/src/main/java/com/example/demo/service/impl/QuestionServiceImpl.java:51]: Are you using this method, checkQuestion and checkTitle methods?

**Review by aleksandar-bujaroski-armedia — APPROVED** [review]

**Pull request merged**